app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+
+  scope :available, -> { where archived_at: nil }
```

Comment:
中括弧 `{}` の左右も半角スペース付けない方が（私の好み＆）Sun* 標準だったかと
小括弧 `()` の左右に半角スペースを付けないのと揃えた方がしっくり来ます
```suggestion
  scope :available, ->{ where archived_at: nil}
```